lib/LocalSigner.ts

```diff
@@ -20,7 +21,17 @@ export default class LocalSigner implements ISigner {
   }
 
   public async sign (header: object, content: object): Promise<string> {
-    const compactJws = await secp256k1.ES256K.sign(content, this.privateKey, header);
+    const key = await Secp256k1KeyPair.from({
+      type: 'JsonWebKey2020',
+      publicKeyJwk: this.privateKey,
```

Comment:
Assigning to `privateKey` here seems not right to do. Which leads to a bigger question, what's wrong with the original one liner? Is it that `@transmute/did-key-secp256k1` is being deprecated?

Reply:
https://github.com/decentralized-identity/ion-sdk/pull/20/commits/d63d081902df1691bc1f9b1c27f4d204e43d7519

Yes this lib was old, see https://github.com/decentralized-identity/ion-sdk/pull/20#discussion_r758438788